gm4_shiny_items/pack.mcmeta

```diff
@@ -0,0 +1,26 @@
+{
+    "pack": {
+        "pack_format": 5,
+        "description": "A Gamemode 4 Module"
+    },
+    "module_name": "Shiny Items",
+    "module_id": "shiny_items",
+  "creation_meta":"Created using the Gamemode 4 Module Template Generator (gm4.co/modules/generator)",
+    "site_description": "This module allows the player to apply the enchantment glow to most items in the game. This can be used by players to draw attention to items they find special and want to show off.",
```

Comment:
Site description are the short blurbs displayed beneath a module's icon. This will probably break the site / appear really small.